rllib/connectors/connector.py

```diff
@@ -106,7 +106,7 @@ def in_eval(self):
     def __str__(self, indentation: int = 0):
         return " " * indentation + self.__class__.__name__
 
-    def to_state(self) -> Tuple[str, List[Any]]:
+    def to_state(self) -> Tuple[str, Any]:
```

Comment:
Can you add a doc on what str and Any should be?

Reply:
done